sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_healthcare.py

```diff
@@ -42,7 +42,8 @@ def test_all_successful_passing_dict(self, client):
         docs = [{"id": "1", "language": "en", "text": "Patient does not suffer from high blood pressure."},
                 {"id": "2", "language": "en", "text": "Prescribed 100mg ibuprofen, taken twice daily."}]
 
-        response = client.begin_analyze_healthcare(docs, show_stats=True, polling_interval=self._interval()).result()
+        poller = client.begin_analyze_healthcare_entities(docs, show_stats=True, polling_interval=self._interval())
+        response = poller.result()
 
         self.assertIsNotNone(response.statistics)
 
```

Comment:
looking at 
```
for doc in response:
            self.assertIsNotNone(doc.id)
            self.assertIsNotNone(doc.statistics)
            self.assertIsNotNone(doc.entities)
            self.assertIsNotNone(doc.relations)
```
Does that mean that the relations are not inside an entity? but globally per document?

Reply:
Looking at `HealthcareEntity` [here](https://github.com/Azure/azure-sdk-for-python/blob/eed2d38e3d5405aac603b61890962c8ee12d037f/sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py#L380) it seems like `Length` is also missing. If the idea is to add `length` in other PR, then ignore my comment

Reply:
Are we not just using the `len` function (or whatever is natural for each language) for this?  I hadn't actually planned for this work, so some extra context on why we are doing this would be helpful.  Is it just for Healthcare or for everything?

Reply:
because of the exposure of StringIndexType the user can specify other type that is not the default for Python. if Python can determine the len() based on the specific encoding then it is not needed.
This is not true for .NET so we do have to expose `Length` everywhere

Reply:
Ok, thanks for the clarification.  I might need to open a new PR for this one then, as it sounds like we might need to write a helper method to determine this.  I will have to do some research to see what needs to be done.  

As for your first question above, I haven't changed anything about how the relations are being represented, so right now it's just a list of relations with references to the entities objects in the `entities` array.  I expect there to be a lot of change coming in the next API version so I haven't done the same thing that Java/JS are doing with graph building yet.

Reply:
I think the work should be the same to what you do today to expose Offset. @iscai-msft  for guidance on Python :)

> relations

I see. if Python is ok with that, then we are all good. I wanted to call it out just in case